staging/src/k8s.io/client-go/rest/plugin.go

```diff
@@ -30,8 +30,8 @@ type AuthProvider interface {
 	// WrapTransport allows the plugin to create a modified RoundTripper that
 	// attaches authorization headers (or other info) to requests.
 	WrapTransport(http.RoundTripper) http.RoundTripper
-	// Login allows the plugin to initialize its configuration. It must not
-	// require direct user interaction.
+	// Login allows the plugin to initialize its configuration. It may involve
```

Comment:
This is a major change that hasn't been discussed.

Reply:
Other auth providers already require user interaction. e.g. the azure auth-n plugin can prompt you.

Reply:
So is the comment on the interface wrong or the azure implementation wrong?

Reply:
The Login command has never been implemented, the Azure transport wrapper does this, so I'd call the comment is outdated.

Conversation about interactive prompts here https://github.com/kubernetes/kubernetes/pull/43987#issuecomment-297867514